### PR TITLE
init textures

### PR DESCRIPTION
**Pull request by hadavidson**
Added stock minecraft textures, will be working to expand to all mods. I added a section in the main readme on setup

### CODE REVIEW

**Comment by checkraisefold** [comment]
enjoy getting fucked by mojang licensing
in all seriousness its a really bad idea to upload all the vanilla textures, find a texture pack that has a decent license so you can use it

**Comment by checkraisefold** [comment]
that works too

File: README.md [hunk 1]
```diff
@@ -46,9 +46,11 @@
 ✓ Add name to code, and make a UI
 
 # HOW TO USE TEXTURES
-1. Extract frontend/public/texture.zip
-2. this should extract as /frontend/public/textures/minecraft/(all textures in here)
-3. Profit
+1. Go to your minecraft install location
+2. go to versions then 1.12.2 and open the .jre and an archive
```
Comment (Vap0r1ze) [inline README.md:50]: You probably meant "as an archive"

**Review by LostGameDev — APPROVED** [review]
yes

**Pull request closed without merging**